workflows/src/main/kotlin/com/r3/corda/lib/tokens/workflows/flows/rpc/EvolvableTokens.kt

```diff
@@ -0,0 +1,97 @@
+package com.r3.corda.lib.tokens.workflows.flows.rpc
+
+import co.paralleluniverse.fibers.Suspendable
+import com.r3.corda.lib.tokens.contracts.states.EvolvableTokenType
+import com.r3.corda.lib.tokens.workflows.flows.evolvable.CreateEvolvableTokensFlow
+import com.r3.corda.lib.tokens.workflows.flows.evolvable.CreateEvolvableTokensFlowHandler
+import com.r3.corda.lib.tokens.workflows.flows.evolvable.UpdateEvolvableTokenFlow
+import com.r3.corda.lib.tokens.workflows.flows.evolvable.UpdateEvolvableTokenFlowHandler
+import com.r3.corda.lib.tokens.workflows.flows.evolvable.maintainers
+import com.r3.corda.lib.tokens.workflows.flows.evolvable.otherMaintainers
+import com.r3.corda.lib.tokens.workflows.flows.evolvable.participants
+import com.r3.corda.lib.tokens.workflows.flows.evolvable.subscribersForState
+import net.corda.core.contracts.StateAndRef
+import net.corda.core.contracts.TransactionState
+import net.corda.core.flows.FlowLogic
+import net.corda.core.flows.FlowSession
+import net.corda.core.flows.InitiatedBy
+import net.corda.core.flows.InitiatingFlow
+import net.corda.core.flows.StartableByRPC
+import net.corda.core.identity.AbstractParty
+import net.corda.core.identity.Party
+import net.corda.core.transactions.SignedTransaction
+
+@InitiatingFlow
+@StartableByRPC
+class CreateEvolvableTokens<T : EvolvableTokenType>
```

Comment:
CreateEvolvableToken or Tokens?

Reply:
Do we need the type param here?

Reply:
aha, i didn't do rebase ;)

Reply:
Also, tokens, because we create many. For update i left updating only one token, so it is UpdateToken, but let me know if you prefer other naming

Reply:
ok makes sense - cheers!